client/ayon_houdini/plugins/publish/collect_render_products.py

```diff
@@ -166,6 +166,15 @@ def get_aov_identifier(self, render_product):
             str: The AOV identifier
 
         """
+        # Allow explicit naming through custom attribute on the render product
+        aov_identifier = render_product.GetPrim().GetAttribute(
+            "ayon:aov_identifier").Get()
+        if aov_identifier:
+            self.log.debug(
+                "Using explicit ayon:aov_identifier on render product"
+                " '{render_product}': {aov_identifier}")
```

Comment:
There's a missing `f` here
```suggestion
                f" '{render_product}': {aov_identifier}")
```